stories/ui/RecommendedBadge.js

```diff
@@ -3,8 +3,11 @@ import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { fakeI18n } from 'tests/unit/helpers';
 
-import RecommendedBadge from 'ui/components/RecommendedBadge';
+import { RecommendedBadgeBase } from 'ui/components/RecommendedBadge';
 
 storiesOf('RecommendedBadge', module).add('default', () => (
-  <RecommendedBadge i18n={fakeI18n({ includeJedSpy: false })} />
+  <RecommendedBadgeBase
+    size="large"
```

Comment:
Should we have a story for `size="small"` too?

Reply:
Yes, there wasn't much in this story to begin with so I wasn't sure how much time I'd have to build it out. It needs to show icons without badges, too. I'll see if I can do something quick.

Reply:
I added stories for both components